src/NServiceBus.Core/CircuitBreakers/RepeatedFailuresOverTimeCircuitBreaker.cs

```diff
@@ -28,7 +29,7 @@ public void Success()
             Logger.InfoFormat("The circuit breaker for {0} is now disarmed", name);
         }
 
-        public void Failure(Exception exception)
+        public Task Failure(Exception exception)
```

Comment:
It is internal. So no



-- 
*#NSBcon 2015 • The "All About NServiceBus" conference is back *
*• Learn more <http://go.particular.net/l/27442/2015-08-21/5d4wzp> →*